lib/get-config.js

```diff
@@ -39,7 +40,9 @@ async function requirePlugin(workingDir, pluginName, fromConfigPath) {
   let pluginPath = resolve.sync(pluginName, { basedir, extensions: ['.js', '.mjs', '.cjs'] });
 
   try {
-    const { default: plugin } = await import(pluginPath);
+    let pluginURL = pathToFileURL(pluginPath);
```

Comment:
Note: this definitely fixes a bug in plugin resolution, but only for windows users (we previously attempted to `import('c:/foo/bar/baz.js')` which is not allowed, you have to do `import('file:///c:/foo/bar/baz.js')`).

Reply:
This should be labeled as a `bug fix` then right?